L/ldid/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder
 
 name = "ldid"
-version = v"2.1.3" # <-- Fake version to build for new platforms
+version = v"2.1.4" # <-- Fake version to build with new OpenSSL compat bounds
```

Comment:
The upstream repo has newer versions: https://git.saurik.com/ldid.git. BTW, why do we fork it?

Reply:
It’s actually forked from https://github.com/tpoechtrager/ldid which made a bunch of build system changes

Reply:
To actually answer your question, the `tpoechtrager` fork doesn't have any new commits, and glancing over the changes made in Saurik's source, I'm not sure it's worth my time to try and integrate them.  They don't seem that applicable to us.